src/Microsoft.Extensions.DotnetToolDispatcher.Sources/project.json

```diff
@@ -3,7 +3,7 @@
   "shared": "*.cs",
   "dependencies": {
     "Microsoft.DotNet.Cli.Utils": "1.0.0-*",
-    "Microsoft.Extensions.PlatformAbstractions": "1.1.0-*",
+    "Microsoft.Extensions.PlatformAbstractions": "1.0.0-*",
```

Comment:
We should remove these dependencies entirely. Dependencies listed in shared src projects don't get added to the consuming project.